easybuild/tools/options.py

```diff
@@ -607,9 +607,12 @@ def easyconfig_options(self):
         # easyconfig options (to be passed to easyconfig instance)
         descr = ("Options for Easyconfigs", "Options to be passed to all Easyconfig.")
 
-        opts = None
+        opts = OrderedDict({
+            'inject-checksums': ("Inject checksums of specified type for sources/patches into easyconfig file(s)",
+                                 'choice', 'store_or_None', CHECKSUM_TYPE_SHA256, CHECKSUM_TYPES),
+        })
         self.log.debug("easyconfig_options: descr %s opts %s" % (descr, opts))
-        self.add_group_parser(opts, descr, prefix='easyconfig')
```

Comment:
Otherwise the option would be `--easyconfig-inject-checksums`, which is overkill imho.

There were no options in this category yet, so this change is fine.